### PR TITLE
DynamicLoader + LibELF: Remove some allocations using StringView literals 

### PR DESCRIPTION
**Pull request by bgianfo**

### CODE REVIEW

File: Userland/Libraries/LibELF/DynamicLinker.cpp [hunk 1]
```diff
@@ -99,7 +99,7 @@ static Result<NonnullRefPtr<DynamicLoader>, DlErrorMessage> map_library(const St
 
 static Result<NonnullRefPtr<DynamicLoader>, DlErrorMessage> map_library(const String& name)
 {
-    if (name.contains("/")) {
+    if (name.contains("/"sv)) {
```
Comment (alimpfard) [inline Userland/Libraries/LibELF/DynamicLinker.cpp:102]: This could also be just a character, but meh

Comment (bgianfo) [reply Userland/Libraries/LibELF/DynamicLinker.cpp:102]: I checked last night, there is no `String::contains()` for just a char? Maybe it's names something else?

Comment (alimpfard) [reply Userland/Libraries/LibELF/DynamicLinker.cpp:102]: :thinking:
```c++
    [[nodiscard]] bool contains(char) const;
```

Comment (alimpfard) [reply Userland/Libraries/LibELF/DynamicLinker.cpp:102]: Oh that's a `String`.
this is from `StringView`

**Pull request merged**